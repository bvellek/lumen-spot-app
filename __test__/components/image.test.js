import React from 'react';
import renderer from 'react-test-renderer';

import Image from '../../public/js/components/image';

describe('<Image />', () => {
  it('it renders correctly', () => {
    const component = renderer.create(
      <Image imageLink='https://testing.com' imageLinkTarget='_testing' src='https://testing.com/photo' alt='test description' aperture='2.8' iso='200' ss='1/200' photo='tester mctesting' />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
