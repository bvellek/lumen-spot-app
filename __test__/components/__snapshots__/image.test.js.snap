// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> it renders correctly 1`] = `
<figure
  className="pic-fig"
>
  <a
    href="https://testing.com"
    target="_testing"
  >
    <img
      alt={undefined}
      src={undefined}
    />
  </a>
  <figcaption>
    <p>
      <span
        className="fig-num"
      >
        A: 
      </span>
      2.8
       
      <span
        className="fig-num"
      >
        ISO: 
      </span>
      200
       
      <span
        className="fig-num"
      >
        SS: 
      </span>
      1/200
    </p>
    <p
      className="photog"
    >
      Â© 
      <span>
         
      </span>
      / 500px
    </p>
  </figcaption>
</figure>
`;
